Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -45,6 +45,7 @@ class GLContextWidget final : public GUI::Frame {
     void set_texture_scale(float scale) { m_texture_scale = scale; }
     void set_texture_enabled(bool texture_enabled) { m_texture_enabled = texture_enabled; }
     void set_mag_filter(GLint filter) { m_mag_filter = filter; }
+    void set_window(RefPtr<GUI::Window> window) { m_window = window; };
```

Comment:
This shouldn’t needed, this widget inherits from `GUI::Widget` which already has an API to access the window (`GUI::Widget::window()`). Does that API not work here for some reason?

Reply:
You are completely correct! I event mentioned it in my PR notes because I felt like something was wrong. It was my clangd/lsp setup that was misconfigured and was complaining about `window()` not being available. 

I've removed everything relating to that.

Big thanks for your help and review!